security/tools/generate_cert/main.go

```diff
@@ -55,6 +55,8 @@ var (
 	outPriv        = flag.String("out-priv", "priv.pem", "Output private key file.")
 	keySize        = flag.Int("key-size", 2048, "Size of the generated private key")
 	mode           = flag.String("mode", selfSignedMode, "Supported mode: self-signed, signer, citadel")
+	//enable this flag if istio mTLS is enabled and the service is running as server side
```

Comment:
Suggest changing "//enable this flag ..." to "// Enable this flag ..."

Reply:
done